### PR TITLE
(Variable --->Tensor) refine the OP doc for API2.0 

### PR DESCRIPTION
**Pull request by wangchaochaohu**
### PR types

Function optimization
### PR changes

OPs
### Describe

API2.0文档规范中 
1修改alias的格式
2. Variable--->Tensor
3.polish the doc 

eye
image

image

full
image
image

full_like

image
image
index_select

image
image

linspace
image
image

concat
image

image

### CODE REVIEW

File: python/paddle/fluid/layers/tensor.py [hunk 1]
```diff
@@ -633,56 +630,55 @@ def assign(input, output=None):
...
 
     Returns:
-        Variable: Tensor which is created according to shape and dtype.
+        Tensor: Tensor which is created according to shape and dtype.
 
     Raises:
         TypeError: The dtype must be one of bool, float16, float32, float64, int32 and int64
-            and the data type of out Tensor must be the same as the dtype. 
-        TypeError: The shape must be one of list, tuple and Variable.
+            and the data type of out Tensor must be the same as the ``dtype``.
```
Comment (lanxianghit) [inline python/paddle/fluid/layers/tensor.py]: the data type of out Tensor must be the same as the ``dtype``. 

这里的out Tensor指的是out这个参数指定的tensor吧？应该不是返回的output？如果这样的话，直接写``out``，指明与参数的关系

File: python/paddle/fluid/layers/tensor.py [hunk 2]
```diff
@@ -633,56 +630,55 @@ def assign(input, output=None):
...
           data2 = fluid.layers.fill_constant(shape=[2,1], value=5, dtype='int64', out=data1)
           # data1=[[5], [5]] data2=[[5], [5]]
 
-          # attr shape is a list which contains Variable Tensor.
+          # attr shape is a list which contains Tensor.
           positive_2 = fluid.layers.fill_constant([1], "int32", 2)
           data3 = fluid.layers.fill_constant(shape=[1, positive_2], dtype='float32', value=1.5) # data3=[[1.5, 1.5]]
 
-          # attr shape is an Variable Tensor.
+          # attr shape is an Tensor.
```
Comment (lanxianghit) [inline python/paddle/fluid/layers/tensor.py]:
```suggestion
          # attr shape is a Tensor.
```

File: python/paddle/fluid/layers/tensor.py [hunk 3]
```diff
@@ -633,56 +630,55 @@ def assign(input, output=None):
...
           shape = fluid.layers.fill_constant([2], "int32", 2) # shape=[2,2]
           data4 = fluid.layers.fill_constant(shape=shape, dtype='bool', value=True) # data4=[[True,True],[True,True]]
           
-          # attr value is an Variable Tensor.
+          # attr value is an Tensor.
```
Comment (lanxianghit) [inline python/paddle/fluid/layers/tensor.py]:
```suggestion
          # attr value is a Tensor.
```

File: python/paddle/fluid/layers/tensor.py [hunk 4]
```diff
@@ -1070,23 +1069,32 @@ def zeros(shape, dtype, force_cpu=False, name=None):
...
+            and the data type of out Tensor must be the same as the dtype. 
+        TypeError: The ``shape`` must be one of list, tuple and Tensor. The data type of ``shape`` must
+            be int32 or int64 when it's a Tensor.
     Examples:
         .. code-block:: python
 
           import paddle.fluid as fluid
           data = fluid.layers.zeros(shape=[3, 2], dtype='float32') # [[0., 0.], [0., 0.], [0., 0.]]
+          
+          #shape is a Tensor
```
Comment (lanxianghit) [inline python/paddle/fluid/layers/tensor.py]:
```suggestion
          # shape is a Tensor
```

File: python/paddle/tensor/creation.py [hunk 5]
```diff
@@ -153,7 +157,7 @@ def ones(shape, dtype=None, name=None):
           # [[1 1]
           #  [1 1]]
           
-          #shape is a Variable
+          #shape is a Tensor
```
Comment (lanxianghit) [inline python/paddle/tensor/creation.py]:
```suggestion
          # shape is a Tensor
```

File: python/paddle/tensor/creation.py [hunk 6]
```diff
@@ -210,28 +214,45 @@ def ones_like(x, dtype=None, name=None):
...
-          data = paddle.zeros(shape=[2, 2], dtype='int32', name='zeros') # [[0, 0], [0, 0]]
+          data = paddle.zeros(shape=[3, 2], dtype='float32') 
+          # [[0. 0.]
+          #  [0. 0.]
+          #  [0. 0.]]
+          data = paddle.zeros(shape=[2, 2]) 
+          # [[0. 0.]
+          #  [0. 0.]]
+          
+          #shape is a Tensor
```
Comment (lanxianghit) [inline python/paddle/tensor/creation.py]:
```suggestion
          # shape is a Tensor
```

File: python/paddle/tensor/search.py [hunk 7]
```diff
@@ -138,27 +138,27 @@ def argmax(input, axis=None, dtype=None, out=None, keepdims=False, name=None):
...
 
     Returns:
-        Variable: A Tensor with same data type as `input`.
+        Tensor: A Tensor with same data type as ``x``.
     
     Raises:
-        TypeError: x must be a Variable and the dtype of x must be one of  float32, float64, int32 and int64.
-        TypeError: index must be a Variable adn the dtype of index must be int32 or int64.
+        TypeError: ``x`` must be a Tensor and the data type of ``x`` must be one of  float32, float64, int32 and int64.
+        TypeError: ``index`` must be a Tensor adn the data type of ``index`` must be int32 or int64.
```
Comment (lanxianghit) [inline python/paddle/tensor/search.py]:
```suggestion
        TypeError: ``index`` must be a Tensor and the data type of ``index`` must be int32 or int64.
```

**Review by lanxianghit — DISMISSED** [review]
LGTM

File: python/paddle/tensor/creation.py [hunk 8]
```diff
@@ -327,29 +352,30 @@ def eye(num_rows, num_columns=None, dtype=None, name=None):
...
     Returns:
-        Variable: Tensor which is created according to shape and dtype.
+        Tensor: Tensor which is created according to ``shape``, ``fill_value`` and ``dtype``.
 
     Raises:
-        TypeError: The `dtype` must be one of None, bool, float16, float32, float64, int32 and int64.
-        TypeError: The `shape` must be one of Variable, list and tuple.
+        TypeError: The ``dtype`` must be one of None, bool, float16, float32, float64, int32 and int64.
+        TypeError: The ``shape`` must be one of Variable, list and tuple. the data type of ``shape`` must
+            be int32 or int64 when the it's a Tensor
```
Comment (swtkiwi) [inline python/paddle/tensor/creation.py:378]: Variable -> Tensor

Comment (wangchaochaohu) [reply python/paddle/tensor/creation.py:378]: Done

**Review by swtkiwi — APPROVED** [review]
LGTM

**Review by lanxianghit — APPROVED** [review]
LGTM

**Pull request merged**